src/jumio-kyc/fixtures/workflow.ts

```diff
@@ -153,7 +155,7 @@ export const WORKFLOW_RETRIEVE_FIXTURE = ({
             documentNumber: '1111111',
             optionalMrzField1: 'Z11',
             optionalMrzField2: '11111111',
-            currentAge: '70',
+            currentAge: age,
```

Comment:
is this provided on every document or is it optional?

Reply:
API says every document, but there are other fields that haven't been there when it was a nonoptional return, I will guard for this case.